src/jbi/bugzilla_objects.py

```diff
@@ -0,0 +1,245 @@
+"""
+Bugzilla Typed Objects for ease of use throughout JBI
+"""
+import datetime
+from typing import Dict, List, Optional, Tuple
+from urllib.parse import ParseResult, urlparse
+
+from pydantic import BaseModel  # pylint: disable=no-name-in-module
+
+
+class BugzillaWebhookUser(BaseModel):
+    """Bugzilla User Object"""
+
+    id: int
+    login: str
+    real_name: str
+
+
+class BugzillaWebhookEventChange(BaseModel):
+    """Bugzilla Change Object"""
+
+    field: str
+    removed: str
+    added: str
+
+
+class BugzillaWebhookEvent(BaseModel):
+    """Bugzilla Event Object"""
+
+    action: str
+    time: Optional[datetime.datetime] = None
+    user: Optional[BugzillaWebhookUser] = None
+    changes: Optional[List[BugzillaWebhookEventChange]] = None
+    target: Optional[str] = None
```

Comment:
Good question; I believe the `= None` was because of FastAPI's use of Optional's and how to present them in the swagger docs. Since we're not using it as a Response, Request, Body, Query, or Path object in a FastAPI sense; then I believe the `= None` is not a requirement of Pydantic and is just additional bloat-text.